backend/src/installerservice/installerservice.service.ts

```diff
@@ -189,4 +188,20 @@ export class InstallerService {
             return (accessUrl && accessUrl !== '') ? 'INSTALLED' : 'INSTALL_FAILED';
         }
     }
+
+    private async getHelmChartsPath(rootPath: string, chartConfigDataPath: string) {
+        let helmChartPath: string;
+        if (chartConfigDataPath) {
+            helmChartPath = rootPath + '/' + chartConfigDataPath;
+        } else {
+            const results = search.recursiveSearchSync('Chart.yaml', rootPath);
```

Comment:
maybe we can change it to an async call

Reply:
Recommend to using sync version for search single file.